src/Hooks/VaporWorkCommandFinishedListener.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Laravel\Nightwatch\Hooks;
+
+use Illuminate\Console\Events\CommandFinished;
+use Laravel\Nightwatch\Core;
+use Laravel\Nightwatch\State\CommandState;
+use Throwable;
+
+/**
+ * @internal
+ */
+final class VaporWorkCommandFinishedListener
+{
+    /**
+     * @param  Core<CommandState>  $nightwatch
+     */
+    public function __construct(
+        private Core $nightwatch,
+    ) {
+        //
+    }
+
+    public function __invoke(CommandFinished $event): void
+    {
+        try {
+            if ($event->command === 'vapor:work') {
+                $this->nightwatch->digest();
+            }
```

Comment:
I think we can remove this conditional. We only register it when the command is vapor:work, so it should never evaluate to false.

Reply:
True! Updated in 8e72ec48e6d1832d30fb945a8c0300cc3e2dc9c0